### ISSUE TITLE
what about ait

### ISSUE THREAD

**Issue by Boom-Hacker** [evshiron/rocm_lab#15]
i only ran aitemplate in navi3_rel_ver_1.0,it is so old

**Comment by evshiron** [comment]
Yes, that branch is very old. I made random fixes while debugging and only managed to bring it to a point where it can achieve a score of 25it/s. According to reports, using [this commit of ROCm LLVM](https://github.com/RadeonOpenCompute/llvm-project/commit/1f2f539f7cab51623fad8c8a5b574eda1e81e0c0) can reach 30it/s.

The submodule in this branch is linked to the specified branch of Composable Kernel, which has a Fused Attention implementation for Navi 3x.

I spent a lot of time trying to integrate this Fused Attention into PyTorch before. And you can find my efforts here:

* https://github.com/orgs/are-we-gfx1100-yet/repositories

If you're interested, you can check out the repos in the org to conduct further research.
